pkg/gpio/gpio_linux.go

```diff
@@ -39,6 +40,45 @@ func (v Value) String() string {
 	return "1"
 }
 
+// GetPinID computes the sysfs pin ID for a specific port on a specific GPIO
+// controller chip. It gets the base offset of the chip, and adds the specific
+// pin number.
```

Comment:
add: The controller arg is matched to a gpiochip's label in sysfs.

Reply:
done